generate-matrix

```diff
@@ -14,3 +14,7 @@ csv_file=matrix_$(date +%F_%T).csv
 ods_file=matrix_$(date +%F_%T).ods
 
 scp content-store-1.integration:/var/apps/content-store/dump.csv ./$csv_file
+
+echo Removing dump and dump script from integration
+
+ssh content-store-1.integration 'rm ./dump-data && rm ./$csv_file'
```

Comment:
Ooh, I've just noticed I'm not sure if this will work since it needs to remove it from the `/var/apps/content-store` folder which I don't think it goes to by default when you SSH in.